migration/v0.18.x-v0.19.x/opensearch-migration-run.sh

```diff
@@ -71,6 +84,9 @@ fi
 echo "---
 "
 
+# BEGIN TAKE SNAPSHOT
```

Comment:
Does it make sense to also check that there are no snapshots currently in progress?